api-private.swagger.yaml

```diff
@@ -208,14 +198,45 @@ paths:
           description: The JSON was malformed.
         '422':
           description: JSON schema validation failed.
+  '/pdf-export/create-job/{uuid}':
+    post:
+      tags:
+        - pdf export
+      summary: Create a PDF export job.
+      parameters:
+        - name: uuid
+          in: path
+          type: string
+          description: The UUID fo the node to export.
+          pattern: '^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$'
+          required: true
+        - name: latest
+          in: query
+          type: boolean
+          description: Whether to use the latest revision of each page.
+          required: false
+      responses:
+        '200':
+          description: An existing PDF export job was found and returned to the client.
+          schema:
```

Comment:
should be content / mime-type / schema eg:
```
content:
  application/json:
    schema:
      $ref: ...
```

Reply:
Just copying how it's done elsewhere in the file. When I apply your suggested change, I get an error; see the screenshot below:

![image](https://user-images.githubusercontent.com/30697386/64355605-68965600-cff9-11e9-9fe5-479c56102392.png)

I've noticed that this particular file uses version 2 of the Swagger Specification, while the other file uses OAS version 3. Do you think that could be the cause of the error?

Reply:
Ah! Yes, sorry I didn't realize this was still on Swagger 2. Let's leave it as you have it for now and update it to OAS 3 as a separate task.

Reply:
[ITB-1022](https://io1dev.atlassian.net/browse/ITB-1022) assigned to @jmorahan.